### ISSUE TITLE
Re-enable Global Visual Focus in theme Calydon

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4453]
Now that we have theme `calydon` in place let's try to re-enable the global visual focus in the `calydon/cx-base` before trying to fix it component by component.

**Comment by developpeurweb** [comment]
To QA this ticket, theme `calydon` needs to be activated. All elements with implicit focus will show a blue outline as they're focused on. (buttons, inputs, checkboxes, radio buttons, links).

Individual inconsistencies will be dealt with in this other board:
https://github.com/SAP/cloud-commerce-spartacus-storefront/projects/40?card_filter_query=label%3Aa11y%2Fux%2Ffocus

**Comment by KateChuen** [comment]
- Find a store -> View all stores, select a store, then we click on 'Get Directions'. can't really see the blue outline on the 'Get Directions' button because the button turns black when clicked -> Ticket ?
